website/docs/defaultplatform.md

```diff
@@ -19,6 +21,28 @@ Premake 5.0.0 alpha 12 or later.
 ### Examples ###
 
 ```lua
-defaultplatform "value"
+workspace "MyWorkspace"
+  configurations { "Debug", "Release" }
+  platforms { "Static32", "Shared32", "Static64", "Shared64" }
+  defaultplatform "Shared64" -- Default platform from "Static32" to "Shared64"
+
+  filter "platforms:Static32"
+    kind "StaticLib"
+    architecture "x32"
+
+  filter "platforms:Static64"
+    kind "StaticLib"
+    architecture "x64"
+
+  filter "platforms:Shared32"
+    kind "SharedLib"
+    architecture "x32"
+
+  filter "platforms:Shared64"
+    kind "SharedLib"
+    architecture "x64"
+	
 ```
+### See Also ###
 
+* [defaultplatform](defaultplatform.md)
```

Comment:
Oh no, I am so sorry! 😅 I asked you to add:
```
* [defaultplatform](defaultplatform.md)
```

but of course what I meant was:
```
* [platforms](platforms.md)
```

Please do this one last fix 😄

You can also add `* [defaultplatform](defaultplatform.md)` to [platforms.md](https://github.com/premake/premake-core/blob/master/website/docs/platforms.md) so these pages link each other as they are related.